### PR TITLE
#2300 - Update institution hyperlinks

### PR DESCRIPTION
**Pull request by dheepak-aot**
- [x] Update the hyperlink in login page
image

 - [x] Update the Schedule D hyperlink in designation agreement page
image

### CODE REVIEW

**Review by sh16011993 — APPROVED** [review]
👍

File: sources/packages/forms/src/form-definitions/designationagreementdetails.json [hunk 1]
```diff
@@ -3718,7 +3717,7 @@
                           "value": ""
                         }
                       ],
-                      "content": "<a href=\"https://studentaidbc.ca/sites/all/files/school-officials/Schedule_D_Designation_Agreement.pdf\" target=\"_blank\">Schedule D - Information Sharing agreement</a>",
+                      "content": "<a href=\"https://studentaidbc.ca/sites/all/files/school-officials/sims-information-sharing-agreement.pdf\" target=\"_blank\">Schedule D - Information Sharing agreement</a>",
```
Comment (andrewsignori-aot) [inline sources/packages/forms/src/form-definitions/designationagreementdetails.json:3720]: Maybe not part of the PR but for security recommendations, we usually add `rel="noopener noreferrer"` but apparently designation agreement and some others are missing it.
That said, since they are Ministry links I do not see an issue.

**Review by andrewsignori-aot — APPROVED** [review]
👍

File: sources/packages/forms/src/form-definitions/designationagreementdetails.json (hunk 1 above)
Comment (dheepak-aot) [reply sources/packages/forms/src/form-definitions/designationagreementdetails.json:3720]: Good point. I would prefer to create a ticket to check all the URLs for this because even in the same form, schedule B and schedule A doesn't have `rel="noopener noreferrer"`.

**Review by guru-aot — APPROVED** [review]
LGTM, nice work @dheepak-aot

**Review by ann-aot — APPROVED** [review]
👍

**Comment by HRAGANBC** [comment]
Reviewed and signed off by Hugh on October 12. Everything A-OK. Thanks
